pkg/kritis/metadata/containeranalysis/containeranalysis.go

```diff
@@ -107,23 +120,19 @@ func (c Client) fetchVulnerabilityOccurrence(containerImage string, kind string)
 		return nil, fmt.Errorf("%s is not a valid image hosted in GCR", containerImage)
 	}
 
-	req := &grafeas.ListOccurrencesRequest{
-		Filter:   fmt.Sprintf("resourceUrl=%q AND kind=%q", util.GetResourceURL(containerImage), kind),
-		PageSize: constants.PageSize,
-		Parent:   fmt.Sprintf("projects/%s", getProjectFromContainerImage(containerImage)),
-	}
+	req := createListOccurrencesRequest(containerImage, kind)
 
 	it := c.client.ListOccurrences(c.ctx, req)
```

Comment:
Did the test actually fail with the old code? I can see that the grafeas client implements some pagination logic, so it would be great to verify that this patch is necessary.

Reply:
This test still passes with the old code. There may be some benefit in using the new code, as it makes it more explicit that the pages are being "flipped" and that you have received all the results when the `nextPageToken` is blank.

Reply:
I think the page flipping logic is already implemented in https://github.com/grafeas/kritis/blob/588f70e090ad2ee4c98e65a6d105976ade4aabb2/vendor/cloud.google.com/go/containeranalysis/apiv1beta1/grafeas_v1_beta1_client.go#L181
So it's actually desirable that the details is abstracted away (hidden from this function here).

Reply:
Done.